ext/fg/js/frontend.js

```diff
@@ -272,6 +272,16 @@ class Frontend {
         return this._lastShowPromise;
     }
 
+    _updateTextScannerEnabled() {
+        const enabled = (
+            this.options !== null &&
```

Comment:
I think this null check should probably be removed and `setDisabledOverride` called after prepare in preview frame.

Reply:
bd717c087652dce4955f587d2e545419bf992a16